.github/workflows/trivy.yml

```
@@ -0,0 +1,43 @@
+name: Trivy
+
+on:
+  pull_request:
+  push:
+    branches: [ main ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)